.github/workflows/packages.yaml

```diff
@@ -0,0 +1,51 @@
+name: Submit Packages
+on:
+  release:
+    types: [published]
+
+env:
+  CODER_VERSION: "${{ github.event.release.tag_name }}"
+
+jobs:
+  winget:
+    runs-on: windows-latest
+    steps:
+      - name: Install wingetcreate
+        run: |
+          Invoke-WebRequest https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
+
+      # the package version is the same as the release tag without the leading
+      # "v", and with a trailing ".0" (e.g. "v1.2.3" -> "1.2.3.0")
+      - name: Calculate package version
+        id: version
+        run: |
+          $version = $env:CODER_VERSION -replace "^v", ""
+          $version += ".0"
```

Comment:
Why are you using a trailing 0? Usually, the coder releases have 3-digit version numbers.

Reply:
The NSIS installer required me to add the extra 4th section to the version number, so it shows up in Windows settings app as `0.11.0.x`. Not sure if that's a Windows requirement or an NSIS requirement, but for consistency between winget and the installed package on Windows I opted to keep these the same.

I changed the windows installer build script to add `.0` instead of the date as the date was actually being clipped off funnily enough as it's not a 16 bit number